src/components/BorrowerProfile/LoanStory.vue

```diff
@@ -206,7 +211,7 @@ export default {
 			this.isoCode = loan?.geocode?.country?.isoCode ?? '';
 			this.tags = loan?.tags ?? [];
 			this.gender = loan?.gender ?? '';
-			this.sector = loan?.sector ?? '';
+			this.sector = loan?.sector ?? { name: '' };
```

Comment:
Is this change needed with the optional chaining I added?

Reply:
I think it would overwrite the initial value which contains this stub object in `data` above if it came back undefined.